include/onnxruntime/core/framework/execution_provider.h

```diff
@@ -202,21 +202,23 @@ class IExecutionProvider {
 
   /**
      Indicate whether the graph capturing mode (e.g., cuda graph) is enabled for
-     the provider. Currently only CUDA execution provider supports it.
+     the provider. Currently only CUDA/TensorRT/Rocm execution providers support it.
    */
   virtual bool IsGraphCaptureEnabled() const { return false; }
 
   /**
      Indicate whether the graph has been captured and instantiated. Currently
-     only CUDA execution provider supports it.
+     only CUDA/TensorRT/Rocm execution providers support it.
    */
   virtual bool IsGraphCaptured() const { return false; }
```

Comment:
Does this need change to IsGraphCaptured(int id)?

Reply:
Yes, it makes sense to add the graph_id to catch errors earlier